api/mesh/v1alpha1/zone_ingress.proto

```diff
@@ -28,26 +35,26 @@ message ZoneIngress {
 
   message Networking {
     // Address on which inbound listener will be exposed
-    string address = 1;
+    string address = 1 [ (doc.required) = true ];
 
     // AdvertisedAddress defines IP or DNS name on which ZoneIngress is
     // accessible to other Kuma clusters.
-    string advertisedAddress = 2;
+    string advertisedAddress = 2 [ (doc.required) = true ];
 
     // Port of the inbound interface that will forward requests to the service.
-    uint32 port = 3;
+    uint32 port = 3 [ (doc.required) = true ];
 
     // AdvertisedPort defines port on which ZoneIngress is accessible to other
     // Kuma clusters.
-    uint32 advertisedPort = 4;
+    uint32 advertisedPort = 4 [ (doc.required) = true ];
```

Comment:
same optional

Reply:
it's not: https://github.com/kumahq/kuma/blob/e2f56dda41a08bd3743bcccbf99d937fdd0403f8/pkg/core/resources/apis/mesh/zone_ingress_validator.go#L25-L27